### DISCUSSION TITLE
1-bit full adder combinational circuit

### DISCUSSION THREAD

**Discussion by visionvlsi** [The-OpenROAD-Project/OpenROAD-flow-scripts#920] [Q&A] [answered]
Please help me,
why is that I am unable to run the flow on 1-bit full adder. I am following correct directory structure and my config file contents are:

export DESIGN_NICKNAME = fa
export DESIGN_NAME = fa
export PLATFORM    = sky130hd
export VERILOG_FILES = ./designs/src/$(DESIGN_NICKNAME)/fa.v
export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
export DIE_AREA = 0 0 100 100
export CORE_AREA = 5 5 90 90

Since constraints file is must I just kept in constraint.sdc
_____________
current_design fa
create_clock -period 100 -name virtual_clock

**Comment by rovinski** [comment, marked as answer]
It's impossible to know without knowing the error. Please post your log.

**Reply by vijayank88** [reply]
@rovinski This is the error
![image](https://user-images.githubusercontent.com/85787913/227533414-d9a962b3-ad36-42af-bd7b-e60ad3ffba41.png)

**Reply by maliberty** [reply]
The error is tell you that the block is to small to fit any met4 grid.  You should increase the area of the block (or reduce the power grid).

**Reply by visionvlsi** [reply]
That is correct. How do I do either of the two reasons stated? Please let me know, I wanna try that.

**Comment by maliberty** [comment]
See https://openroad-flow-scripts.readthedocs.io/en/latest/user/FlowVariables.html
in particular the floorplan variables.  You can either set the area with DIE_AREA/CORE_AREA or CORE_UTILIZATION/CORE_MARGIN
